lib/MockRequest.js

```diff
@@ -29,6 +30,15 @@ MockRequest.prototype.url = function() {
 };
 
 /**
+ * Get/set the parsed URL query
+ * @returns {Object}
+ */
+MockRequest.prototype.query = function() {
```

Comment:
I don't like how the query data will be duplicated between `.url()` and `.query()`.... but I can see how its useful having it accessible as a map